lib/index.js

```diff
@@ -207,7 +207,8 @@ function decodeSingle (parsedType, data, offset) {
   var size, num, ret, i
 
   if (parsedType.name === 'address') {
-    return decodeSingle(parsedType.rawType, data, offset)
+    var decoded = decodeSingle(parsedType.rawType, data, offset)
+    return utils.setLengthLeft(decoded, 20)
```

Comment:
Since `rawType` here is `uint160`, the `decoded` is an instance of bn.js. Because of that this could use the more optimised `toString('le', 20)`.

Can you also add tests for this?